templates/puppetdb-deployment.yaml

```diff
@@ -111,7 +111,7 @@ spec:
             - name: PUPPETSERVER_PORT
               value: "{{ template "puppetserver.puppetserver-masters.port" . }}"
             - name: DNS_ALT_NAMES
-              value: {{ template "puppetdb.fullname" . }}
++              value: {{ template "puppetdb.san" . }}
```

Comment:
I think we have here one space more than required.

Reply:
It was an actual plus sign, I've pushed a commit fixing it.